content/OATH/YKOATH_Protocol.adoc

```diff
@@ -198,6 +199,41 @@ Resets the application to just-installed state.
 |Success | 0x9000
 |================
 
+== RENAME INSTRUCTION
+Renames a credential.
+
+NOTE: This is only available in YubiKey 5.3 and newer.
+
+=== Request Syntax
+|===================
+|CLA |0x00
+|INS |0x05
+|P1  |0x00
+|P2  |0x00
+|Lc  |Length of Data
+|Data|Rename Data
+|===================
+
+=== Rename Data
+|===================
+|Old Name tag    |0x71
+|Old Name length |Length of old name data, max 64 bytes
+|Old Name data   |Old name
+|New Name tag    |0x71
+|New Name length |Length of new name data, max 64 bytes
+|New Name data   |New name
+|===================
+
+=== Response Codes
+|=========================
+|Success          | 0x9000
+|No such object   | 0x6984
+|Name already used| 0x6984
```

Comment:
2 errors have the same error code?

Reply:
Yes.

Reply:
It seems overload from client side that we need to distinct between 2 errors

Reply:
ok, since it's postfact documentation, my comments can't affect any change, but I wanted to double check that it's actually not typos